### PR TITLE
Removes some problematic names from random names generation

### PR DESCRIPTION
**Pull request by Haliris**
## About The Pull Request

Replaces some names in the pool used by random generation that in some configuration could lead to bwoinks or reports.

## Why It's Good For The Game

Avoids unecessary admin overhead while also ensuring players aren't led to breaking rules through the random name generator. 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
Random names still generate on locale.

Can't really bring the negative proof that the names in question don't generate anymore.

</details>

## Changelog
:cl:
tweak: tweaked the string/names files to prevent some bad names from being randomly generated.
/:cl:

### CODE REVIEW

File: strings/names/last.txt [hunk 1]
```diff
@@ -528,7 +528,7 @@ Warren
 Watson
 Webb
 Weeter
-Weinstein
```
Comment (EvilDragonfiend) [inline strings/names/last.txt:531]: how bad is this?

Comment (CydiaLamiales) [reply strings/names/last.txt:531]: It just references a very famous american film producer, he worked on all 3 LOTR movies.

Comment (Haliris) [reply strings/names/last.txt:531]: Harvey is a first name. I favoured modifying last names for simplicity's sake.

File: strings/names/first_male.txt [hunk 2]
```diff
@@ -234,14 +234,14 @@ Fernando
...
+Flinn
 Flint
-Flip
 Francis
 Francisco
 Frank
 Frankie
 Franklin
-Fridge
+Fritz
```
Comment (Tsar-Salat) [inline strings/names/first_male.txt]: uhh pretty sure this is a filtered word

Comment (Tsar-Salat) [reply strings/names/first_male.txt]: or maybe it was just filtered on Bay and I'm getting my wires crossed

**Review by Rukofamicom — APPROVED** [review]
I don't see any issues or have any complaints about the names being changed.

File: strings/names/first_male.txt (hunk 2 above)
Comment (Haliris) [reply strings/names/first_male.txt]: It shouldn't be filtered afaik, I only know that Fritz is a very deprecated and ancient way to refer to German people in France. However I don't really like it, and the fact we had this conversation is reason enough for me to change it.

**Comment by Haliris** [comment]
I'll make another push later to change some other names that have been pointed out to me after I made the PR (Fritz and Goodman). I started this when we noticed some really problematic names that could come out of generation (the clown one for instance), but drifted a bit towards changing names that could raise admins' eyebrows.

File: strings/names/first_male.txt (hunk 2 above)
Comment (PowerfulBacon) [reply strings/names/first_male.txt]: If there is any doubt it could cause issues, remove it

**Comment by Rukofamicom** [comment]
I must ask for curiosity's sake what is wrong with the name "Scott"

File: strings/names/clown.txt [hunk 3]
```diff
@@ -74,12 +74,12 @@ Moinen
 Mr Shoe
 Mr Weird
 Ningelo
+Palazzo Perfect
 Patches
 Pepinpop
 Pocket
 Pogo
 Polite Pablo
-Post Nose
```
Comment (Archanial) [inline strings/names/clown.txt:82]: Why?

File: strings/names/first.txt [hunk 4]
```diff
@@ -11,7 +11,7 @@ Addison
 Addyson
 Adeline
 Aden
-Adolph
```
Comment (Archanial) [inline strings/names/first.txt:14]: It's actual human name...

File: strings/names/last.txt [hunk 5]
```diff
@@ -173,9 +173,9 @@ Garrison
 Gettemy
 Gibson
 Glover
-Goebbles
-Goodman
-Graham
```
Comment (Archanial) [inline strings/names/last.txt:178]: Why Graham in particular?

File: strings/names/last.txt [hunk 6]
```diff
@@ -232,7 +232,7 @@ Ironmonger
 Isaman
 Isemann
 Ivanov
-Jackson
```
Comment (Archanial) [inline strings/names/last.txt:235]: Jackson? From what I understand it seems very common.

File: strings/names/last.txt [hunk 7]
```diff
@@ -386,11 +386,11 @@ Pinney
 Poehl
 Poley
 Polson
-Potter
+Potlay
 Powell
 Power
 Powers
-Pratt
```
Comment (Archanial) [inline strings/names/last.txt:393]: So is Pratt

File: strings/names/last.txt [hunk 8]
```diff
@@ -441,11 +441,11 @@ Sandys
 Sauter
 Saylor
 Schaeffer
+Schomer
 Scherer
 Schmidt
 Schofield
-Schrader
-Scott
```
Comment (Archanial) [inline strings/names/last.txt:448]: Why?

**Review by Archanial — DISMISSED** [review]

**Review by Crossedfall — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
image

If a name has the potential to cause issues, it can go. There is no harm in replacing names with other names.

**Pull request merged**